### PR TITLE
Add tip: git log -n

### PR DESCRIPTION
**Pull request by shfshanyue**
I really like this project and I'm glad to contribute to it. Cheers guys!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -777,6 +778,17 @@ git status --ignored
 git log Branch1 ^Branch2
 ```
 
+## List N last commits
```
Comment (hemanth) [inline README.md]: let us use `n` instead?

Comment (shfshanyue) [inline README.md]: ok!

Comment (shfshanyue) [inline README.md]: It's ok.

**Comment by hemanth** [comment]
Thanks.

**Pull request merged**